libs/partners/box/pyproject.toml

```diff
@@ -0,0 +1,97 @@
+[tool.poetry]
+name = "langchain-box"
+version = "0.1.0"
+description = "An integration package connecting Box and LangChain"
+authors = []
+readme = "README.md"
+repository = "https://github.com/langchain-ai/langchain"
+license = "MIT"
+
+[tool.poetry.urls]
+"Source Code" = "https://github.com/langchain-ai/langchain/tree/master/libs/partners/box"
+
+[tool.poetry.dependencies]
+python = ">=3.9.0,<3.13"
+langchain-core = "^0.2.0"
+langchain-community = "^0.2.0"
+box-sdk-gen = {extras = ["jwt"], version = "^1.1.0"}
+unstructured = {extras = ["all-docs"], version = "^0.15.1"}
+nltk = "3.8.1"
+pillow = "10.4.0"
+pillow-heif = "0.18.0"
+matplotlib = "3.9.2"
```

Comment:
similar - what are these used for?

Reply:
These all showed as dependencies that were unresolved when testing UnstructuredImageLoader before I found [all-docs]. Let me take a look at langchain-unstructured. Agree it would be better to not have langchain-community dependencies. When I looked at their documentation, it showed these loaders in community.